homeassistant/components/toon/const.py

```diff
@@ -51,61 +51,45 @@
         ATTR_NAME: "Boiler Module Connection",
         ATTR_SECTION: "thermostat",
         ATTR_MEASUREMENT: "boiler_module_connected",
-        ATTR_INVERTED: False,
         ATTR_DEVICE_CLASS: DEVICE_CLASS_CONNECTIVITY,
-        ATTR_ICON: None,
         ATTR_DEFAULT_ENABLED: False,
     },
     "thermostat_info_burner_info_1": {
         ATTR_NAME: "Boiler Heating",
         ATTR_SECTION: "thermostat",
         ATTR_MEASUREMENT: "heating",
-        ATTR_INVERTED: False,
-        ATTR_DEVICE_CLASS: None,
         ATTR_ICON: "mdi:fire",
         ATTR_DEFAULT_ENABLED: False,
     },
     "thermostat_info_burner_info_2": {
         ATTR_NAME: "Hot Tap Water",
         ATTR_SECTION: "thermostat",
         ATTR_MEASUREMENT: "hot_tapwater",
-        ATTR_INVERTED: False,
-        ATTR_DEVICE_CLASS: None,
         ATTR_ICON: "mdi:water-pump",
-        ATTR_DEFAULT_ENABLED: True,
     },
     "thermostat_info_burner_info_3": {
         ATTR_NAME: "Boiler Preheating",
         ATTR_SECTION: "thermostat",
         ATTR_MEASUREMENT: "pre_heating",
-        ATTR_INVERTED: False,
-        ATTR_DEVICE_CLASS: None,
         ATTR_ICON: "mdi:fire",
-        ATTR_DEFAULT_ENABLED: False,
```

Comment:
`False` is not the default for `ATTR_DEFAULT_ENABLED`. Should we remove this?

Reply:
Hmm yeah, that should be in a separate PR. I Will revert this part.